usr/src/uts/common/fs/zfs/sys/uberblock_impl.h

```diff
@@ -54,6 +54,11 @@ struct uberblock {
 
 	/* highest SPA_VERSION supported by software that wrote this txg */
 	uint64_t	ub_software_version;
+
+	/* These fields are reserved for features that are under development: */
+	uint64_t	ub_mmp_magic;
+	uint64_t	ub_mmp_delay;
+	uint64_t	ub_checkpoint_txg;
```

Comment:
Please add a third field for MMP, `uint64_t ub_mmp_sequence`, placed after `ub_mmp_delay`.  We will use it to provide sub-second resolution to allow faster detection of foreign imports.